### DISCUSSION TITLE
用支付宝续费成功之后提示账户到期

### DISCUSSION THREAD

**Discussion by chenrenying** [getlantern/lantern-forum-cn#611] [General]
已经用支付支成功了，但账户提示将到期，联系之后未回复。请帮我续费。邮箱79958798@qq.com
订单金额

617.00元(84.44美元)
汇率
1美元 = 7.3066人民币
支付时间
2024-06-12 16:33:14

商品说明
NEVENETWORKS.COM

订单号
2024061222001385521409745579
商家订单号
src_1PQmbVHGmABNLbfYoy2MBkq7
![Screenshot_2024-06-13-10-47-35-897_com eg android AlipayGphone](https://github.com/getlantern/lantern-forum-cn/assets/173225717/1f12970b-8035-4953-8aea-54d9fc107018)
![mmexport1718247734945](https://github.com/getlantern/lantern-forum-cn/assets/173225717/6eec26b7-737e-4d0d-8507-b0adf21a3e22)
![mmexport1718768190435](https://github.com/getlantern/lantern-forum-cn/assets/173225717/0a80a3d1-1745-4b9c-8181-d3e5eee61ed0)

**Comment by MaxMKMax** [comment]
@chenrenying https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.

**Comment by Chency1213** [comment]
我的也快过期了,我都不知道在什么地方续费支付宝

**Reply by MaxMKMax** [reply]
@Chency1213 您好。我可以知道为什么您无法续订，因为您没有看到付款选项，或者是否存在阻止您付款的错误或付款限制? 您介意告诉我确切的问题并为我提供一些屏幕截图吗？太感谢了。

**Comment by Chency1213** [comment]
没有信用卡, 请问有其他付款方式吗?&nbsp;



&nbsp;


Nancy
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Mike ***@***.***&gt;; 
发送时间: 2024年8月2日(星期五) 凌晨0:43
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [getlantern/lantern-forum-cn] 用支付宝续费成功之后提示账户到期 (Discussion #611)














 
@Chency1213 您好。我可以知道为什么您无法续订，因为您没有看到付款选项，或者是否存在阻止您付款的错误或付款限制? 您介意告诉我确切的问题并为我提供一些屏幕截图吗？太感谢了。

**Comment by Chency1213** [comment]
已支付了, ***@***.***,&nbsp; 请麻烦需要
附件是支付宝的支付凭证.
订单号2024092422001302011429835216
商家订单号&nbsp;src_1Q20hKHGmABNLbfYNgaEq8pG






Nancy
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月2日(星期五) 凌晨0:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 用支付宝续费成功之后提示账户到期 (Discussion #611)






@Chency1213 您好。我可以知道为什么您无法续订，因为您没有看到付款选项，或者是否存在阻止您付款的错误或付款限制? 您介意告诉我确切的问题并为我提供一些屏幕截图吗？太感谢了。

**Comment by Chency1213** [comment]
已支付了, ***@***.***,&nbsp; 请麻烦需要
附件是支付宝的支付凭证.
订单号2024092422001302011429835216
商家订单号 src_1Q20hKHGmABNLbfYNgaEq8pG






Nancy
***@***.***



&nbsp;




------------------ 原始邮件 ------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月2日(星期五) 凌晨0:43
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 用支付宝续费成功之后提示账户到期 (Discussion #611)






@Chency1213 您好。我可以知道为什么您无法续订，因为您没有看到付款选项，或者是否存在阻止您付款的错误或付款限制? 您介意告诉我确切的问题并为我提供一些屏幕截图吗？太感谢了。
